cmd/convox/templates/init/rails/docker-compose.yml

```diff
@@ -6,3 +6,5 @@ web:
   ports:
     - 80:4000
     - 443:4001
+  volumes:
```

Comment:
I've just done some overdue testing with Docker for Mac on a pretty heavy-duty Rails app and sadly found the volume performance to be terrible.

There's a pretty interesting thread on the topic in the Docker forums here: https://forums.docker.com/t/file-access-in-mounted-volumes-extremely-slow-cpu-bound/8076/15 

Seems like the osxfs performance is not really up to snuff yet. There are some alternatives presented in there and also some benchmark data.

The last word from Docker was 11days ago:

> Just an update here from the Docker camp...
We're still working on it! We've added some small performance boosts over the last few Beta's and performance is heading in the right direction. There are a few other fixes that should be heading in over the next few betas that will also provide a decent boost. The target is to be much faster than VirtualBox and we're confident we can do this before release.
>
> In the meantime the best thing you can do to help is to give us samples of workloads you feel are slow ideally in a a form that we can replicate e.g Dockerfiles, Compose Files etc... We can then analyse these and try to optimise as much as possible for each!
>
> Thanks for bearing with us and for trying the Docker for Mac beta